shared/package.json

```diff
@@ -204,6 +205,7 @@
     "electron": "2.0.5",
     "electron-download": "git://github.com/keybase/electron-download#keybase-fixes-off-410",
     "electron-packager": "12.1.0",
+    "electron-rebuild": "^1.8.2",
```

Comment:
do we need to call this now?

Reply:
🤔Maybe not. I'll drop it for now.

Reply:
🤔 Maybe so. I needed to run again after updating Node. Are you able to build this branch without first running `./node_modules/.bin/electron-rebuild`?